### ISSUE TITLE
If the user specifies a test set fit, the entire training set fit should be used for training

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google-code-export/wnd-charm#11]
```

wndchrm still uses the default split ratio of 0.75 even when the user specifies 
a second .fit file for testing.

workaround: specify -r1.0, ex:
wndchrm test -r1.0 training_set.fit training_set.fit
```

Original issue reported on code.google.com by `christop...@nih.gov` on 18 Feb 2011 at 6:41
